openedx_learning/apps/authoring/components/api.py

```diff
@@ -603,3 +606,54 @@ def _error_header(error: AssetError) -> dict[str, str]:
     )
 
     return HttpResponse(headers={**info_headers, **redirect_headers})
+
+
+def set_collections(
+    learning_package_id: int,
+    component: Component,
+    collection_qset: QuerySet[Collection],
```

Comment:
@navinkarkera One question: What is the reason to use a QuerySet here? I think it is cleaner and easier for developers using this API to use a list of IDs here.

Reply:
@ChrisChV I don't have a strong reason except for maintaining consistency with other APIs, for example: [add_to_collection](https://github.com/open-craft/openedx-learning/blob/83aec7f07203d52818e1eed0c513dc244129580c/openedx_learning/apps/authoring/collections/api.py#L125-L131) and [remove_from_collection](https://github.com/open-craft/openedx-learning/blob/83aec7f07203d52818e1eed0c513dc244129580c/openedx_learning/apps/authoring/collections/api.py#L161-L165) accept entities queryset.

Reply:
Thanks!